### PR TITLE
fix cs to avoid deprecation in php 8.3

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterBase.php [hunk 1]
```diff
@@ -117,7 +117,7 @@ protected function dispatchMany(array $nodes): void
      *
      * @return object|array - PHPCR QOM object or array of objects
      */
-    public function dispatch(AbstractNode $node)
+    public function dispatch(QBConstants $node)
```
Comment (SenseException) [inline lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterBase.php]: `AbstractNode` seems to be in the same namespace. Neither `use` is needed nor the alias.

File: lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterPhpcr.php [hunk 2]
```diff
@@ -290,7 +290,7 @@ protected function walkOperandStaticLiteral(OperandStaticLiteral $node): Literal
     {
         $value = $node->getValue();
 
-        if ($field = $node->getParent()?->getChildOfType(AbstractNode::NT_OPERAND_DYNAMIC)) {
+        if ($field = $node->getParent()?->getChildOfType(QBConstants::NT_OPERAND_DYNAMIC)) {
```
Comment (SenseException) [inline lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterPhpcr.php:293]: `AbstractNode` seems to be in the same namespace. Neither use is needed nor the alias.

File: lib/Doctrine/ODM/PHPCR/Query/Builder/QueryBuilder.php [hunk 3]
```diff
@@ -28,7 +28,7 @@
  *
  * @author Daniel Leech <daniel@dantleech.com>
  */
-class QueryBuilder extends AbstractNode
+class QueryBuilder extends QBConstants
```
Comment (SenseException) [inline lib/Doctrine/ODM/PHPCR/Query/Builder/QueryBuilder.php:31]: Same here. Did you chose to use an alias for a reason? A renaming in a future major version maybe?

**Review by SenseException — CHANGES_REQUESTED** [review]

File: lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterBase.php (hunk 1 above)
Comment (dbu) [reply lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterBase.php]: that was the cs fixer. i think someone originally aliased AbstractNode to QBConstants for using the constants, while keeping AbstractNode as typehint... changed it all to AbstractNode now.

**Pull request merged**